Update pull_request_template.md (#2)
CliMA/.github: pull_request_template.md

- appropriate docstrings on all functions, structs, and modules, and included relevent documentation.

-->

----

> - [ ] I certify that I have read the review checklist and complied accordingly.

Reviewer: ```suggestion
- [ ] I have read and checked the items on the review checklist.
```
